### PR TITLE
[4.3] Update workflow config

### PR DESCRIPTION
**Pull request by wouterj**
Rescued some really bad git stuff from me while merging: https://github.com/symfony/symfony-docs/pull/11580

Original description:

> Hi,
> 
> This draft PR follows this one
> https://github.com/symfony/symfony-docs/pull/11492/files (config for 3.X version)
> but for new version, mostly 4.3
> 
> I checked this PR also
> https://github.com/symfony/symfony/pull/30890, @lyrixx review appreciated please
> 
> Maybe to merge after merging https://github.com/symfony/symfony-docs/pull/11578, and with a rebase ?
> 
> --------------------
> 
> Draft for now
> 
> - Closes https://github.com/symfony/symfony-docs/issues/11229
> 
> this one can be closed also:
> 
> - Closes https://github.com/symfony/symfony-docs/issues/9475
> - Closes https://github.com/symfony/symfony-docs/issues/10197

### CODE REVIEW

File: components/workflow.rst [hunk 1]
```diff
@@ -79,10 +79,11 @@ Usage
 -----
 
 When you have configured a ``Registry`` with your workflows,
-you can retrieve a workflow from it and use it as follows::
+you can retreive a workflow from it and use it as follows::
```
Comment (alexislefebvre) [inline components/workflow.rst]: It looks like `retrieve` is the right orthography.

File: components/workflow.rst [hunk 2]
```diff
@@ -79,10 +79,11 @@ Usage
...
 
     // ...
     // Consider that $blogPost is in place "draft" by default
+    // Consider that $post is in state "draft" by default
```
Comment (alexislefebvre) [inline components/workflow.rst]: The variable `$post` doesn't exist.

**Comment by OskarStark** [comment]
Would you send a PR?

File: components/workflow.rst (hunk 2 above)
Comment (noniagriconomie) [reply components/workflow.rst]: yes i updated it in another PR, and on the original one i added this line
which then create a fail merge :/

**Pull request merged**